internalsdk/android.go

```diff
@@ -58,22 +57,33 @@ type Settings interface {
 	TimeoutMillis() int
 }
 
+// AdSettings is an interface for retrieving mobile ad settings from the global config
+type AdSettings interface {
```

Comment:
@atavism Are we still using this anywhere? I didn't see it anywhere. If not can we just remove it?